### DISCUSSION TITLE
Autoload for "Small Truck"

### DISCUSSION THREAD

**Discussion by SSlz** [ddewar/FS22_UniversalAutoloadModhubAddon#65] [Non-Modhub Mod Support]
https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=254525&title=fs2022

This is the link to the vehicle. can you also integrate an autoload here?

**Comment by ddewar** [comment]
Yes, I will get this in ASAP. I'll keep you updated.

**Reply by SSlz** [reply]
great, thank you very much
